dotcom-rendering/index.d.ts

```diff
@@ -873,14 +873,7 @@ interface ComponentNameChunkMap {
 	addWhen: BlockElementType;
 }
 
-interface InteractiveBlockLoadable extends ComponentNameChunkMap {
-	chunkName: 'InteractiveBlockComponent';
-	addWhen: InteractiveBlockElement['_type'];
-}
-
 // There are docs on loadable in ./docs/loadable-components.md
```

Comment:
Do we still need the comment if the corresponding type definition has been removed?

Reply:
Good catch! Deleting this